ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/concurrency/main/module.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.concurrency">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+    <resources>
+        <!-- TODO preview module will instead pack org.wildfly:wildfly-concurrency-impl-31 -->
+        <artifact name="${org.wildfly:wildfly-concurrency-impl-30}"/>
+        <artifact name="${org.wildfly:wildfly-concurrency-spi}"/>
+    </resources>
+    <dependencies>
+        <module name="io.smallrye.jandex"/>
+        <module name="jakarta.annotation.api"/>
+        <module name="jakarta.enterprise.concurrent.api"/>
+        <module name="jakarta.interceptor.api"/>
+        <module name="jakarta.security.jacc.api"/>
```

Comment:
@emmartins There are a lot of deps here, so I wanted to confirm what's truly required.  So I sent up #18337.

I believe what you have here is correct, except jakarta.annotation.api, jakarta.interceptor.api and jakarta.security.jacc.api are not required. CI on my PR will confirm.

As I noted in the description of #18337, I believe moving wildfly-concurrency-spi out of this module will let us prune another set of modules from this to-be-duplicated file.

I'm ok with merging this as is and using my PR to continue to iterate, or if you want to take in some or all of the #18337 change here, that's ok too. When we get to the point of having a 2nd copy of this file is when it's more important to narrow things down.

Reply:
You can merge #18337 instead, and then lets look at how to improve the jboss modules setup. On that I think we should continue with everything out of ee jboss module, since that give us full flexibility in future changes, like a standalone extension/subsystem, so I would vote on breaking up concurrency into 2 modules.

Thanks for the reviews and cleanup.